client/requestUtil.js

```diff
@@ -169,11 +199,216 @@ RequestUtil.prototype.list = function (category, startAt, maxRecords) {
   }
   if (startAt) { options.StartAfter = `${prefix}/${startAt}` }
   return this.withRetry(() => {
-    return s3Helper.listObjects(this.s3, options, !!maxRecords)
+    if (!startAt || startAt === 0 || ((new Date()).getTime() - startAt) >
```

Comment:
minor: `startAt === 0` is unnecessary since `!0` is true

Reply:
thanks, you are right